src/app/accounts/index.ts

```diff
@@ -24,3 +25,26 @@ export const getBusinessMapMarkers = async () => {
   const accounts = AccountsRepository()
   return accounts.listBusinessesForMap()
 }
+
+export const toWalletIds = async (
```

Comment:
Just a minor thing - I think we could find a better name here - perhaps `filteredWalletIds` or something like that.

Reply:
Will include that in the new conversion